testapps/testapp/src/main/res/raw/msal_config.json

```diff
@@ -13,7 +13,7 @@
     }
   ],
   "telemetry": {
-    "pii_enabled": true,
+    "pii_enabled": false,
```

Comment:
Should this be configured in the default app config? Is this off by default?

Reply:
I think we should disable pii by default even for test apps. Because this is a public repo on github. I will change this back for now. And you guys can have a follow-up tmr or before ga.